exporter/loggingexporter/config.go

```diff
@@ -15,28 +15,87 @@
 package loggingexporter // import "go.opentelemetry.io/collector/exporter/loggingexporter"
 
 import (
+	"fmt"
+
 	"go.uber.org/zap/zapcore"
 
 	"go.opentelemetry.io/collector/config"
+	"go.opentelemetry.io/collector/config/configtelemetry"
+	"go.opentelemetry.io/collector/confmap"
 )
 
 // Config defines configuration for logging exporter.
 type Config struct {
 	config.ExporterSettings `mapstructure:",squash"` // squash ensures fields are correctly decoded in embedded struct
 
 	// LogLevel defines log level of the logging exporter; options are debug, info, warn, error.
+	// Deprecated: Use `Verbosity` instead.
 	LogLevel zapcore.Level `mapstructure:"loglevel"`
 
+	// Verbosity defines the logging exporter verbosity.
+	Verbosity configtelemetry.Level `mapstructure:"verbosity"`
+
 	// SamplingInitial defines how many samples are initially logged during each second.
 	SamplingInitial int `mapstructure:"sampling_initial"`
 
 	// SamplingThereafter defines the sampling rate after the initial samples are logged.
 	SamplingThereafter int `mapstructure:"sampling_thereafter"`
+
+	// warnLogLevel is set on unmarshaling to warn users about `loglevel` usage.
+	warnLogLevel bool
```

Comment:
i know that because is private the mapstructure most likely does not unmarshal it, but prefer safe than sorry, can you add a `mapstructure:-` (I think that is the annotation to ignore a field).

Reply:
This does not ignore the field: https://go.dev/play/p/yTXlhIa4IEq it only works with `encoding/json`, but mapstructure does not ignore such a field

Reply:
Should I still add it, if only for consistency?